### PR TITLE
fix: conflict code staging

### PR DESCRIPTION
**Pull request by GarryLaly**
# Helio PR Checklist

## Please review all of the following checks before making PR

- [x] Jika menambahkan sebuah komponen, apakah sudah dipastikan tidak ada komponen yang redundant, baik secara UI maupun fungsional?
- [x] Periksa apakah komponen yang dibuat reusable (contoh: membuat komponen View untuk membuat Divider antar section)
- [x] Periksa untuk penggunaan state management yang baik dan benar (contoh: Context, useState), terutama penggunaan local state untuk optimistic UI (Pengoperan state dan data melalui props dari parent ke children dan sebaliknya)
- [x] Pastikan komponen yang dibuat sudah sesuai dengan module yang ada terutama penamaan dan penempatan untuk meningkatkan readability dan maintainability.
- [x] Implementasi fallback error untuk menangani kesalahan tak terduga, mencegah aplikasi mengalami crash
- [x] Evaluasi kompleksitas kode dan perbaiki logika yang kompleks dengan memecahnya menjadi fungsi atau komponen yang lebih kecil dan lebih mudah dimengerti.
- [x] Apakah sudah resolve semua komentar dari CodeRabbit🐇?
- [x] Apakah semua unit tests sudah Pass/Success? (Tidak ada bypass)
- [x] Periksa semua perubahan teks Bahasa Inggris approved oleh Bastian?
- [x] Apakah sudah menggunakan normalized.dimen/ normalizeFontSize untuk semua perubahan style?
- [x] Pastikan untuk mengikuti [guidelines pembuatan komponen Guidelines About Loading and Layout](https://docs.google.com/document/d/1GHbvEtFrQmEQXYBV9dLrPI654n2DbPwS1qjedajLSiw/edit)

### Author's comment

if you have any unchecked items, please explain the reason why.

### Summary by CodeRabbit

---
- Refactor: Removed duplicate import statements in `ChannelListScreenV2/AnonymousChannelListScreen/index.tsx` and `ProfileScreen/elements/BioAndDMSetting.tsx` for cleaner code.
- Style: Updated padding values in `components/Comments/WriteComment.test.js` to use a normalization function, enhancing consistency across the application.
- Test: Improved safety in `components/ReplyComment/index.js` by using optional chaining when accessing `feedId`, reducing potential errors during runtime.
---

### CODE REVIEW

**Comment by bastian-purrer** [comment]
Image description   CodeRabbit



Currently reviewing new changes in this PR...

<details>
<summary>Commits</summary>
Files that changed from the base of the PR and between ff8ee71a1b88429086bb0ab2d2a88f736691c728 and 4c32809af58f5126835ff5ace7ee040070aceaa6 commits.
</details>

<details>
<summary>Files selected (1)</summary>

* src/screens/ChannelListScreenV2/AnonymousChannelListScreen/index.tsx (1)
</details>






---

**Review by bastian-purrer — COMMENTED** [review]
Image description   CodeRabbit

<details>
<summary>Commits</summary>
Files that changed from the base of the PR and between ff8ee71a1b88429086bb0ab2d2a88f736691c728 and 4c32809af58f5126835ff5ace7ee040070aceaa6 commits.
</details>

<details>
<summary>Files selected (1)</summary>

* src/screens/ChannelListScreenV2/AnonymousChannelListScreen/index.tsx (1)
</details>







<details>
<summary>Files skipped from review due to trivial changes (1)</summary>

* src/screens/ChannelListScreenV2/AnonymousChannelListScreen/index.tsx

</details>

<details>
<summary>Review comments generated (0)</summary>

* Review: 0
* LGTM: 0

</details>

---

<details>
<summary>Tips</summary>

### Chat with Image description  CodeRabbit Bot (`@coderabbitai`)
- Reply on review comments left by this bot to ask follow-up questions. A review comment is a comment on a diff or a file.
- Invite the bot into a review comment chain by tagging `@coderabbitai` in a reply.

### Code suggestions
- The bot may make code suggestions, but please review them carefully before committing since the line number ranges may be misaligned. 
- You can edit the comment made by the bot and manually tweak the suggestion if it is slightly off.

### Pausing incremental reviews
- Add `@coderabbitai: ignore` anywhere in the PR description to pause further reviews from the bot.

</details>

**Review by bastian-purrer — COMMENTED** [review]
Image description   CodeRabbit

<details>
<summary>Commits</summary>
Files that changed from the base of the PR and between 4c32809af58f5126835ff5ace7ee040070aceaa6 and 6c52aa24687477c7c925042315c8cacfa1b1a1d9 commits.
</details>

<details>
<summary>Files selected (1)</summary>

* src/screens/ProfileScreen/elements/BioAndDMSetting.tsx (1)
</details>







<details>
<summary>Files skipped from review due to trivial changes (1)</summary>

* src/screens/ProfileScreen/elements/BioAndDMSetting.tsx

</details>

<details>
<summary>Review comments generated (0)</summary>

* Review: 0
* LGTM: 0

</details>

---

<details>
<summary>Tips</summary>

### Chat with Image description  CodeRabbit Bot (`@coderabbitai`)
- Reply on review comments left by this bot to ask follow-up questions. A review comment is a comment on a diff or a file.
- Invite the bot into a review comment chain by tagging `@coderabbitai` in a reply.

### Code suggestions
- The bot may make code suggestions, but please review them carefully before committing since the line number ranges may be misaligned. 
- You can edit the comment made by the bot and manually tweak the suggestion if it is slightly off.

### Pausing incremental reviews
- Add `@coderabbitai: ignore` anywhere in the PR description to pause further reviews from the bot.

</details>

File: src/components/ReplyComment/index.js [hunk 1]
```diff
@@ -95,7 +95,7 @@ const ReplyCommentId = ({
       <ScrollView ref={scrollViewRef} contentContainerStyle={styles.commentScrollView}>
         <View style={styles.containerComment}>
           <ReplyCommentItem
-            feedId={dataFeed.id}
+            feedId={dataFeed?.id}
```
Comment (bastian-purrer) [inline src/components/ReplyComment/index.js:98]: Image description   CodeRabbit

The change from `feedId={dataFeed.id}` to `feedId={dataFeed?.id}` is a good one. It ensures that if `dataFeed` is null or undefined, the code won't throw an error trying to access the `id` property. This improves the robustness of the code.
```diff
- feedId={dataFeed.id}
+ feedId={dataFeed?.id}
```

File: __tests__/components/Comments/WriteComment.test.js [hunk 2]
```diff
@@ -33,19 +34,15 @@ describe('WriteComment should run correctly', () => {
...
+      flexDirection: 'row'
     });
     expect(styles.container).toEqual({
       flex: 1,
       backgroundColor: COLORS.white,
       width: '100%',
-      paddingRight: 20,
-      paddingLeft: 20,
+      paddingRight: dimen.normalizeDimen(16),
+      paddingLeft: dimen.normalizeDimen(16),
```
Comment (bastian-purrer) [inline __tests__/components/Comments/WriteComment.test.js:45]: Image description   CodeRabbit

The changes made here are good for maintaining consistency in the codebase. The padding values are now being set using the `dimen.normalizeDimen()` function, which ensures that these values are normalized and consistent across different screen sizes.
```diff
- paddingRight: 20,
- paddingLeft: 20,
+ paddingRight: dimen.normalizeDimen(16),
+ paddingLeft: dimen.normalizeDimen(16),
```

**Review by bastian-purrer — COMMENTED** [review]
Image description   CodeRabbit

<details>
<summary>Commits</summary>
Files that changed from the base of the PR and between 6c52aa24687477c7c925042315c8cacfa1b1a1d9 and 7539e0cd9d2782e0d85550f2a12604017d72c68c commits.
</details>

<details>
<summary>Files selected (2)</summary>

* __tests__/components/Comments/WriteComment.test.js (2)
* src/components/ReplyComment/index.js (1)
</details>


<details>
<summary>Files ignored due to filter (15)</summary>

* __tests__/components/AnonymousChat/__snapshots__/AnonymousChatHeader.test.tsx.snap
* __tests__/components/AnonymousChat/child/__snapshots__/ChannelAnonymousSubImage.test.tsx.snap
* __tests__/components/AnonymousChat/child/__snapshots__/ChannelPostNotificationMessage.test.tsx.snap
* __tests__/components/AnonymousChat/child/__snapshots__/ChannelTitle.test.tsx.snap
* __tests__/components/AnonymousChat/child/__snapshots__/ChatItemMyTextV2.test.tsx.snap
* __tests__/components/AnonymousChat/child/__snapshots__/ChatItemTargetText.test.tsx.snap
* __tests__/components/Blocking/__snapshots__/SpecificIssue.test.js.snap
* __tests__/components/Button/__snapshots__/ButtonAddTopicPost.test.js.snap
* __tests__/components/DummyLogin/__snapshots__/DummyLogin.test.js.snap
* __tests__/components/WhoToFollow/__snapshots__/WhoToFollowLabel.test.js.snap
* __tests__/components/WhoToFollow/__snapshots__/WhoToFollowUserInfo.test.js.snap
* __tests__/screens/DiscoveryScreenV2/__snapshots__/Search.test.js.snap
* __tests__/screens/FeedScreen/__snapshots__/ContentLink.test.js.snap
* __tests__/screens/FeedScreen/__snapshots__/ContentPoll.test.js.snap
* __tests__/screens/GroupSetting/__snapshots__/GroupSetting.test.js.snap

</details>







<details>
<summary>Review comments generated (3)</summary>

* Review: 3
* LGTM: 0

</details>

---

<details>
<summary>Tips</summary>

### Chat with Image description  CodeRabbit Bot (`@coderabbitai`)
- Reply on review comments left by this bot to ask follow-up questions. A review comment is a comment on a diff or a file.
- Invite the bot into a review comment chain by tagging `@coderabbitai` in a reply.

### Code suggestions
- The bot may make code suggestions, but please review them carefully before committing since the line number ranges may be misaligned. 
- You can edit the comment made by the bot and manually tweak the suggestion if it is slightly off.

### Pausing incremental reviews
- Add `@coderabbitai: ignore` anywhere in the PR description to pause further reviews from the bot.

</details>

**Review by fajarism — APPROVED** [review]

**Pull request merged**